### ISSUE TITLE
!= Operator should output <>

### ISSUE THREAD

**Issue by ianedwardthomas** [emehrkay/Pypher#28]
Hi,

I think the cypher output for the != operator should be <> not != as this is not valid cypher. 

For example,

MATCH.node('p','person').WHERE.p.property('name')  != 'me' ...
should return cypher of
MATCH (p:person) WHERE p.name <> 'me' ...
and not 
MATCH (p:person) WHERE p.name != 'me' ... 

This workaround will of course produce the same result.
MATCH.node('p','person').WHERE.NOT.p.property('name')  != 'me'

**Comment by emehrkay** [comment]
Until it is fixed, you can do 

```python
p.MATCH.node('p', 'person').WHERE.p.property('name').operator('<>', 'me')
```

Should produce:

`MATCH (p:`person`) WHERE p.`name` <> $NEO_09218_0`

**Comment by emehrkay** [comment]
This was fixed with 0.16.2.

Thanks

**Comment by ianedwardthomas** [comment]
Looks good.  Cheers.
